docs/_interface/conf.md

```diff
@@ -0,0 +1,288 @@
+---
+title: The Configuration File
+sort_key: B
+---
+
+Geth can use a configuration file to specify various parameters, instead of using command line arguments. To get a configuration file
+with the defaults, run:
+
+```sh
+geth dumpconfig > configFile
+```
+
+To use a configuration file, run:
+
+```sh
+geth --config configFile
+```
+
+The configuration file uses the [TOML syntax](https://en.wikipedia.org/wiki/TOML). The section names mostly correspond to [the names
+of packages in the Geth source code](https://pkg.go.dev/github.com/ethereum/go-ethereum#section-directories).
+
+
+## Settings in the Configuration File
+
+### Eth
+
+This package is responsible for running the Ethereum protocol. 
+[The configuration object is documented here](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.4/eth/ethconfig#Config)
+
+#### Network Settings
+
+| Setting           | Type         | Meaning                                                                                              |
+| ----------------- | ------------ | ---------------------------------------------------------------------------------------------------- |
+| NetworkId         | uint(64)     | The Chain ID for the network. [Here is a list of possible values](https://chainlist.org/)            |
```

Comment:
Technically, this is the `network_id`, not the `chain_id`. In most cases, it's identical. Perhaps the doc should say, instead. 

```suggestion
| NetworkId         | uint(64)     | The network ID for the network, which in most cases should be identical to the chain id. [Here is a list of possible values](https://chainlist.org/)            |
```